### PR TITLE
Added Baidu support

### PR DESCRIPTION
**Pull request by shashank-sharma**
Fixes: #202 

Added Button with Baidu icon and the scraper is used to scrape Baidu search engine.

screenshot from 2017-09-27 03-39-12

### CODE REVIEW

File: app/scrapers/ask.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def __init__(cls):
         pass
 
     @classmethod
-    def get_page(cls,query):
+    def get_ask_page(cls,query):
```
Comment (AnshulMalik) [inline app/scrapers/ask.py]: Changes should be relevant to the issue you are solving, this should be in another PR

File: app/templates/index.html [hunk 2]
```diff
@@ -59,7 +59,8 @@
                             <button type="submit" value="bing" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/bing_icon.ico') }}" width="30px" alt="Bing Icon"> Bing</button>
                             <button type="submit" value="duckduckgo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/duckduckgo_icon.png') }}" width="30px" alt="Duckduckgo Icon"> DuckDuckGo</button>
                             <button type="submit" value="yahoo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/yahoo_icon.ico') }}" width="30px" alt="Yahoo Icon"> Yahoo</button>
-			    <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
+			                <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
```
Comment (AnshulMalik) [inline app/templates/index.html]: Inconsistent spacing here

File: app/scrapers/ask.py (hunk 1 above)
Comment (shashank-sharma) [reply app/scrapers/ask.py]: @AnshulMalik This error all of sudden was happening and making travis CI test fail that's why I edited it.

File: app/templates/index.html (hunk 2 above)
Comment (shashank-sharma) [reply app/templates/index.html]: I am not sure but please see previous PR correctly because these errors are generating from other persons code. I am happy to correct them but this leads to reading of whole code from top to bottom

Comment (AnshulMalik) [reply app/templates/index.html]: No no, I am just saying that there is an extra tab here

File: app/scrapers/ask.py (hunk 1 above)
Comment (AnshulMalik) [reply app/scrapers/ask.py]: Yep, I think https://github.com/fossasia/query-server/pull/205 will resolve that issue

Comment (shashank-sharma) [reply app/scrapers/ask.py]: Ok, nice Will wait for that merge

**Comment by shashank-sharma** [comment]
@vaibhavsingh97 I'll do that as soon as this #205 gets accepted

**Comment by cclauss** [comment]
This will need rework based on the current scraper.py

**Comment by shashank-sharma** [comment]
@cclauss You are right, I am again waiting for #205 to get merged and then I will do it

File: app/scrapers/ask.py (hunk 1 above)
Comment (cclauss) [reply app/scrapers/ask.py]: My sense is that #205 is no longer blocking progress on this PR.
